src/link/rgblink.1

```diff
@@ -113,6 +122,45 @@ When making a ROM, be careful that not using this is not a replacement for
 .Xr rgbfix 1 Ap s Fl p
 option!
 .El
+.Ss Scrambling algorithm
+The default section placement algorithm tries to minimize the amount of banks used;
+.Dq scrambling
+instead places sections into a given pool of banks, trying to minimize the amount of sections sharing a given bank.
+This is useful to catch broken bank assumptions, such as expecting two different sections to land in the same bank; this is not guaranteed unless both are manually assigned the same bank number.
```

Comment:
```suggestion
This is useful to catch broken bank assumptions, such as expecting two different sections to land in the same bank, which is not guaranteed unless both are manually assigned the same bank number.
```

Reply:
I originally wrote it like this, but found that the semicolon makes it easier for the eye not to get lost in the sentence.

Reply:
Using 'this' twice in the same sentence to refer to different things might not be great for readability. An alternative would be to parenthesize the additional remark: "This is useful to catch broken bank assumptions, such as expecting two different sections to land in the same bank. (This is something that is not guaranteed unless both are manually assigned the same bank number.)".

Reply:
"This is something that is not guaranteed" can just be "That is not guaranteed", which also avoids confusion over the changing usage of "this".

Reply:
Is it better now?

Reply:
Yes. =)